src/Facade/Services/IConsignmentFacadeService.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Linn.Stores.Facade.Services
+{
+    using Linn.Common.Facade;
+    using Linn.Stores.Domain.LinnApps.Consignments;
+    using Linn.Stores.Resources.Consignments;
+    using Linn.Stores.Resources.RequestResources;
+    using System.Collections.Generic;
+
+    public interface IConsignmentFacadeService : IFacadeService<Consignment, int, ConsignmentResource, ConsignmentUpdateResource>
+    {
+        IResult<IEnumerable<Consignment>> GetByRequestResource(ConsignmentsRequestResource resource);
```

Comment:
is this extra method just to let you do filtering on a resource? If so then IFacadeFilterService gets you that

Reply:
there is a bit of logic in there can't see from other uses of IFacadeFilterService how it would help me but might be missing something